api/src/utils/sanitize-query.ts

```diff
@@ -184,7 +184,7 @@ function sanitizeDeep(deep: Record<string, any>, accountability?: Accountability
 				// way of knowing when to keep nesting and when to stop
 				const [parsedKey, parsedValue] = Object.entries(parsedSubQuery)[0];
 				parsedLevel[`_${parsedKey}`] = parsedValue;
-			} else {
+			} else if (isPlainObject(value)) {
```

Comment:
@rijkvanzanten was this related to #10300   ?

Reply:
No. It was to resolve #10092, which was a regression introduced in https://github.com/directus/directus/pull/10236 which was a (wrong) fix to prevent `null` from showing up in the filter structure, causing an error in apply-query.

Reply:
Ok !